doc/dev/developer_guide/tests-integration-tests.rst

```diff
@@ -267,7 +267,7 @@ exit code zero).
 
 This test can be run with::
 
-    $ teuthology-suite --suite rados/singleton/all/admin-socket.yaml fs/ext4.yaml
+    $ teuthology-suite --machine-type mira --suite rados/singleton/all/admin-socket.yaml fs/ext4.yaml
```

Comment:
s/mira/smithi/ ? as mira nodes are being deprecated and not able to run centos8 based tests.

Reply:
substituted in all instances.